src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp

```diff
@@ -400,15 +404,26 @@
         }
     }
 
+    function makeBackGroundColor(){
+        console.log("failOss: ", failOss.toString());
+        for (const failOssGridId of failOss) {
+            $("#list").jqGrid("setRowData", failOssGridId, false, {'background' : "rgba(255, 0, 0, 0.3)"});
+        }
+    }
+
     const OssBulkGridStatusMessageManager = {
         cleanStatusMessages: function () {
             $("#list").jqGrid("getGridParam", "data").forEach(row => {
-                row.status = ""
+                row.msg = ""
```

Comment:
Please rollback `msg` to `status`.
`cleanStatusMessages` function empty status field of Table.